flake/zombienet.nix

```diff
@@ -97,11 +117,26 @@
           program = self'.packages.zombienet-dali-complete;
         };
 
+        zombienet-dali-centauri-a = {
+          type = "app";
+          program = self'.packages.zombienet-dali-centauri-a;
+        };
+
         zombienet-picasso-complete = {
           type = "app";
           program = self'.packages.zombienet-picasso-complete;
         };
 
+        zombienet-log-follow = {
+          program = pkgs.writeShellApplication rec {
+            name = "zombienet-log-follow";
+            text = ''
+              docker exec -it composable-devnet-a-1   bash  -c 'LOG=$(find /tmp/ -name "zombie-*" | head --lines=1)/alice.log && tail --follow $LOG'
```

Comment:
```suggestion
              docker exec -it composable-devnet-a-1 bash -c 'tail --follow /tmp/zombie-*/alice.log'
```

Reply:
perhaps, can we do such ^^^? or are there more than one `zombie-*` directories?

Reply:
there could be many. if you run this locally.